amazon-kinesis-client/src/main/java/software/amazon/kinesis/coordinator/Scheduler.java

```diff
@@ -126,6 +131,9 @@
 
     private final Object lock = new Object();
 
+    private final long executorLogIntervalMillis = TimeUnit.SECONDS.toMillis(30);
```

Comment:
let's make this static variable.

Reply:
Made variable static and moved to DiagnosticEventLogger